### PR TITLE
[12] authorization service

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: docs/migrating_to_12.md
Comment (v-rocheleau) [inline docs/migrating_to_12.md:1]: Worked as expected.

File: lib/authz/docker-compose.authz.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    healthcheck:
+      test: [ "CMD", "pg_isready", "-q", "-d", "${BENTO_AUTHZ_DB}", "-U", "${BENTO_AUTHZ_DB_USER}" ]
+      timeout: 5s
+      interval: 15s  # Faster interval for authz-db, which authz requires to be healthy
+
+networks:
+  authz-net:
+    external: true
+    name: ${BENTO_AUTHZ_NETWORK}
+  authz-db-net:
```
Comment (v-rocheleau) [inline lib/authz/docker-compose.authz.yaml:40]: We might want to add this to adminer's networks for future authz dev

**Review by v-rocheleau — COMMENTED** [review]
Looking good, will re-review further after I do more testing with https://github.com/bento-platform/bento_web/pull/233

File: lib/authz/docker-compose.authz.yaml (hunk 1 above)
Comment (davidlougheed) [reply lib/authz/docker-compose.authz.yaml:40]: done

File: lib/authz/docker-compose.authz.yaml
Comment (v-rocheleau) [inline lib/authz/docker-compose.authz.yaml:1]: minor: add mem_limit to services

**Review by v-rocheleau — APPROVED** [review]
Tested locally with https://github.com/bento-platform/bento_web/pull/233#pullrequestreview-1433697533 , working as expected.

Was able to look into authz-db using adminer with container name as host.

I think someone who knows nginx better than me should take a look at the config templates.

**Pull request merged**